<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Programming_levels</class>
 <widget class="QDialog" name="Programming_levels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="progLevel1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>Level 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="progLevel2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>Level 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="progLevel3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>200</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>Level 3</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
